src/storage/btree_load.h

```diff
@@ -207,21 +207,13 @@ typedef struct btree_root_header BTREE_ROOT_HEADER;
 struct btree_root_header
 {
   BTREE_NODE_HEADER node;
-  int num_oids;			/* Number of OIDs stored in the Btree */
-  int num_nulls;		/* Number of NULLs (they aren't stored) */
-  int num_keys;			/* Number of unique keys in the Btree */
+  long long num_oids;		/* Number of OIDs stored in the Btree */
+  long long num_nulls;		/* Number of NULLs (they aren't stored) */
+  long long num_keys;		/* Number of unique keys in the Btree */
```

Comment:
How about using `UINT64`?

Reply:
When these values are not unique keys, they may be set to -1.

Reply:
then you can use `INT64`.